src/characters/tDec.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  EncryptedTreasureMap,
+  PublicKey,
+  SecretKey,
+} from '@nucypher/nucypher-core';
+import axios from 'axios';
+
+import { fromHexString } from '../utils';
+
+import { Enrico } from './enrico';
+import { tDecDecrypter } from './universal_bob';
+
+async function getTDecConfig(
+  configLabel: string
```

Comment:
What do you think about providing users with fixed configurations via `configLabel`, vs implementing a Builder pattern to give them a bit more flexibility (while still constraining them to prevent misuse)?

Just my curiosity, I'm out of the loop.

Reply:
i can see both being useful. A static config that you can be accessed via a `label` is quite future safe - you know that all your code/servers etc dies the policy is still valid.

Having a builder puts more responsibility on the user to keep the artifcats safe. That's not a bad thing, it gives them far more flexability.

Reply:
From initial feedback, it appears that flexibility trumps simplicity. tDec adopters prefer to tune their own cohort configurations and offer those 'packages' to their adopters and end-users. See nucypher/tdec/issues/1